coderd/database/migrations/000045_workspacebuildname.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE workspace_builds
+    DROP COLUMN name;
```

Comment:
Do we not have policies for keeping old columns around like we do in v1? @coadler

Reply:
nah, i dont think we even support downgrades currently, so i doubt it makes a difference